app/src/main/java/org/simple/clinic/enterotp/EnterOtpUpdate.kt

```diff
@@ -48,7 +48,7 @@ class EnterOtpUpdate(
     val updatedModel = model.loginFinished()
     val loginFailedModel = updatedModel.loginFailed()
```

Comment:
Can we please update this approach in a future PR. We are creating a login failed model and login finished model irrespective of the results.

Reply:
Yes, will do